templates/server.conf.erb

```diff
@@ -125,7 +125,9 @@ push "<%= @redirect_gateway %>"
 push "dhcp-option DNS <%= @dns %>"
 <% end -%>
 
-<% if @domain != '' -%>
+<% if @domain.is_a? Array -%>
+<% @domain.each do |d| -%>push "dhcp-option DOMAIN <%= "#{d}\"\n" %><% end %>
```

Comment:
Sorry for the long delay.
I do not know if the \n is necessary, I just did a copy/paste of the lines number 286 and then adapt the thing.
I agree, that probably an Array(@domain).each ... should be possible, but I prefer to have equal coding styles in a file ...